### ISSUE TITLE
may your help about the gpsnet?

### ISSUE THREAD

**Issue by ZHUXUHAN** [waxnkw/IETrans-SGG.pytorch#3]
dear sir,
i see the Table 1. in your paper, in all plain baseline, the gpsnet's performance is best. i just hope to reproduce this accuracy, can you tell me what should I do and if I can get your log file about this model, if it's still there.
thanks.

**Comment by waxnkw** [comment]
Actually, GPSNet is not the best plain baseline in Table 1. This result is directly adopted from the paper _"Bipartite Graph Network with Adaptive Message Passing for Unbiased Scene Graph Generation"_, which is with a resampling augmentation. Sorry for my carelessness of forgetting to mark this. I will revise this later. 
We only provide a non-resampling version in the current code (`bash cmds/50/gpsnet/predcls/sup/train.sh`), which is around 66.91 for R@100 and 16.04 for mR@100 of the PREDCLS task. To get the resampling code, you can go through https://github.com/SHTUPLUS/PySGG for more tails.

**Comment by Yuqifan1117** [comment]
dear sir,
as your say, without any augmentation strategy, is Transformer the best plain baseline for all PREDCLS, SGCLS, SGDet tasks? And now the current code is the most primitive version for baseline models and if I can know whether the resampling strategy improve the results as reweighting strategy.
thanks

**Comment by waxnkw** [comment]
Considering the F@100, the transformer is the best. For this version of the code, the transformer for PREDCLS can roughly get 65.7 R@100 and 69.6 mR@100.
I do not try a resampling strategy in my work. But from the "Bipartite Graph Network with Adaptive Message Passing for Unbiased Scene Graph Generation" Table 1, GPSNet with resampling is better than the plain one.
![image](https://user-images.githubusercontent.com/34906503/178682045-21baedaa-3d2f-4d84-aef4-67e5ee3b0663.png)

**Comment by Yuqifan1117** [comment]
已收到！

**Comment by ZHUXUHAN** [comment]
thans for your reply for solveing one of my big confusions!

**Comment by ZHUXUHAN** [comment]
> Considering the F@100, the transformer is the best. For this version of the code, the transformer for PREDCLS can roughly get 65.7 R@100 and 69.6 mR@100. I do not try a resampling strategy in my work. But from the "Bipartite Graph Network with Adaptive Message Passing for Unbiased Scene Graph Generation" Table 1, GPSNet with resampling is better than the plain one. ![image](https://user-images.githubusercontent.com/34906503/178682045-21baedaa-3d2f-4d84-aef4-67e5ee3b0663.png)

maybe the mR@100 is wrong?

**Comment by waxnkw** [comment]
Sorry, it is 19.6.
![image](https://user-images.githubusercontent.com/34906503/178718889-294c1d2c-beb1-4788-8fc6-65b60814f9ac.png)
